File: app/src/main/java/com/example/uohih/joowon/retrofit/ApiService.java
package com.example.uohih.joowon.retrofit;


import com.google.gson.JsonObject;

import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Header;
import retrofit2.http.POST;
import retrofit2.http.Path;

/**
 * api
 *
 * date: 2020-09-24
 * author: jieun
 */
public interface ApiService {
//    final String Base_URL = " http://192.168.219.189:3000/process/";
    String Base_URL = "https://joowon12.herokuapp.com/process/";
    String BASE_URL_NAVER_API = "https://openapi.naver.com/";

    @POST("login_p/{method}")
    Call<ResponseBody> LoginProcessService(@Path("method")  String method,  @Body JsonObject jsonBodyString);
    @POST("base")
    Call<ResponseBody> BaseProcessService(@Body JsonObject jsonBodyString);
    @POST("dairyprocess")
    Call<ResponseBody> EmployeeProcessService(@Body JsonObject jsonBodyString);
    @POST("postprocess")
    Call<ResponseBody> PostProcessService(@Body JsonObject jsonBodyString);
    @POST("boardprocess")
    Call<ResponseBody> BoardProcessService(@Body JsonObject jsonBodyString);

    @POST("v1/nid/me")
    Call<ResponseBody> NaverOpenApiService(@Header("Authorization") String accT);

}

File: app/src/main/java/com/example/uohih/joowon/repository/ApiService.kt
package com.example.uohih.joowon.repository

import com.google.gson.JsonObject
import okhttp3.ResponseBody
import retrofit2.Call
import retrofit2.http.*

/**
 * api
 *
 * date: 2020-09-24
 * author: jieun
 */
interface ApiService {
    @POST("sign_p/{method}")
    fun signInProcessService(@Path("method") method: String, @Body jsonBodyString: JsonObject): Call<ResponseBody?>

    @POST("base")
    fun BaseProcessService(@Body jsonBodyString: JsonObject?): Call<ResponseBody?>

    @POST("dairyprocess")
    fun EmployeeProcessService(@Body jsonBodyString: JsonObject?): Call<ResponseBody?>

    @POST("postprocess")
    fun PostProcessService(@Body jsonBodyString: JsonObject?): Call<ResponseBody?>

    @POST("boardprocess")
    fun BoardProcessService(@Body jsonBodyString: JsonObject?): Call<ResponseBody?>

    @POST
    fun NaverOpenApiService(@Url url: String?, @Header("Authorization") accT: String?): Call<ResponseBody?>

    companion object {
        const val Base_URL = "https://joowon12.herokuapp.com/process/"
//        const val Base_URL = "http://10.0.2.2:8080/process/"
        const val BASE_URL_NAVER_API = "https://openapi.naver.com/v1/nid/me"
    }
}